apps/communications/contacts/js/utilities/telemetry.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+/* global LazyLoader */
+/* global AdvancedTelemetryHelper */
+
+(function(exports) {
+
+  var namespace = 'telemetry_gaia_contacts';
+
+  var dependencies = LazyLoader.load(
```

Comment:
The whole idea is that you lazyload the telemetry.js file and just afterwards you use the telemetry capabilities.

It's loading the dependencies in that way, instead of loading them with the previous lazyload of the telemetry.js file to avoid having to put the dependencies outside this file.

I think this way, dependencies are self contained and will help to remove problems related to too much js loaded and forgotten to be removed.